www/docs/main/awesome-trpc.mdx

```diff
@@ -43,6 +43,12 @@ A collection of resources on tRPC.
 | tRPC - iron-session                                                                                  | https://github.com/parkgang/trpc-iron-session                         |
 | trpc-koa-adapter - tRPC adapter for Koa server                                                       | https://github.com/BlairCurrey/trpc-koa-adapter                       |
 
+## 🧰 Tooling
+
+| Description                                                                                          | Link                                                                    |
+| ---------------------------------------------------------------------------------------------------  | ----------------------------------------------------------------------- |
+| Step CI - Open Source framework for API Quality Assurance, which tests tRPC and other paradigms automated and from Open API spec | https://github.com/stepci/stepci        
```

Comment:
```suggestion
## 🧰 Tooling

| Description                                           | Link                             |
| ----------------------------------------------------- | -------------------------------- |
| Step CI - Automated API Testing and Quality Assurance | https://github.com/stepci/stepci |
```

I think the description is a bit long - especially if you read if on mobile the table kinda looks odd. Shortening it down to the slogan you have on the repo
<img width="401" alt="CleanShot 2022-12-06 at 12 16 10@2x" src="https://user-images.githubusercontent.com/51714798/205897239-8131097c-e7c1-4e6a-ae63-2477153af5cc.png">